roles.json

```
@@ -434,7 +415,8 @@
                 "Evaluating new pull requests for quality, API consistency, Astropy coding standards, and appropriateness within the overall astropy ecosystem, in particular for pull requests spanning multiple sub-packages",
                 "Merging Pull Requests that are non-controversial or after reaching out to relevant subpackage maintainers",
                 "Maintain, review, and advocate for useful interaction between multiple sub-packages", 
-                "Perform initial triaging of issues and pull requests",
+                "Perform initial triaging of issues and pull requests, including moving between repositories",
+                "Ensure adequate labeling of issues and pull requests",
```

Maybe just add this if it's effectively subsumed.
```suggestion
                "Ensure adequate labeling of issues and pull requests",
                "Merge non-controversial pull requests as time allows",
```

---

But this is already mentioned a few lines above.